lib/rubocop/cop/rspec/leaky_constant_declaration.rb

```diff
@@ -23,35 +23,51 @@ module RSpec
       #   # bad
       #   describe SomeClass do
       #     OtherClass = Struct.new
-      #     CONSTANT_HERE = 'is also denied'
+      #     CONSTANT_HERE = 'I leak into global namespace'
       #   end
       #
       #   # good
       #   describe SomeClass do
       #     before do
       #       stub_const('OtherClass', Struct.new)
-      #       stub_const('CONSTANT_HERE', 'is also denied')
+      #       stub_const('CONSTANT_HERE', 'I am undefined after the example')
       #     end
       #   end
       #
       # @example
       #   # bad
       #   describe SomeClass do
-      #     class OtherClass < described_class
-      #       def do_something
+      #     class FooClass < described_class
+      #       def double_that
+      #         some_base_method * 2
       #       end
       #     end
+      #
+      #     it { expect(FooClass.new.double_that).to eq(4) }
       #   end
       #
-      #   # good
+      #   # good - sacrificial class, no constant needs to be defined
```

Comment:
The term "sacrificial class" is ok, but I don't know if that's in wide use or not. I usually just refer to this as an "anonymous class" which I think is accurate/common terminology across languages. It could also potentially say both.

Reply:
You are right. I was carried away by the freshness of that term.